.github/workflows/deploy.yml

```diff
@@ -205,3 +210,18 @@ jobs:
       project_type: ui
       azure_app_name: as-fh-sd-admin-ui
     secrets: inherit
+  
+  api-service-mock-hsda-deploy:
```

Comment:
Can we insert this step after `api-service-directory-deploy` and have its `needs` updated for that, and then the `ui-connect-dashboard-deploy` will need the Mock HSDA Deploy to finish - just to maintain the deploy order of APIs then UIs (at least by name in the deploy output) :)

Reply:
Was trying to avoid lots of changes pushing the needs dependencies around, given that we may get this working in parallel in the near future. Thoughts?